src/module/profile.ts

```diff
@@ -0,0 +1,74 @@
+import { PrismaClient } from "@prisma/client";
+import { Elysia } from "elysia";
+import { getUser, getUserId } from "@/utils/user";
+
+const prisma = new PrismaClient();

```

Comment:
Please change to global prisma to prevent memory leaks.

Sample: https://github.com/esc-chula/larngear24-passport-backend/blob/d1afe6a7c0e54b912b98729ffca25c7eb813e299/src/services/auth.ts#L3
```suggestion
```